jdaviz/utils.py

```diff
@@ -139,8 +139,11 @@ def standardize_metadata(metadata):
     dictionary.
     """
     if isinstance(metadata, fits.Header):
-        out_meta = dict(metadata)
-        out_meta[COMMENTCARD_KEY] = metadata.comments
+        try:
+            out_meta = dict(metadata)
+            out_meta[COMMENTCARD_KEY] = metadata.comments
+        except Exception:  # Invalid FITS header
```

Comment:
Note to self: Maybe can grab that header from https://github.com/spacetelescope/dat_pyinthesky/tree/main/jdat_notebooks/IFU_cube_continuum_fit to test this.

Reply:
Nope. I cannot figure out how to extract the bad header entry out with `astropy.io.fits` without fixing it first, which defeats the purpose. By design, `astropy.io.fits` prevents writing out invalid header. Maybe I will just chuck this under `# pragma: no cover`.